packages/@o3r/telemetry/project.json

```diff
@@ -26,12 +26,8 @@
       "options": {
         "main": "packages/@o3r/telemetry/src/public_api.ts",
         "tsConfig": "packages/@o3r/telemetry/tsconfig.build.json",
-        "outputPath": "packages/@o3r/telemetry/dist",
-        "clean": false
```

Comment:
are you sure there is no problem removing that?
there are several tasks writing on `dist` and I remember we had issues in the past where some files were missing on `dist`

Reply:
Sorry I forgot to turn the PR as draft from the beginning.
Yes, it is an issue for the cache re-hydration, the problem is when the `compile` task is re-run but not the `builder` ones.
I got issue with wrong cache-miss and through it was because of it but after investigation it was mainly due to `schemas`. So I rolled back this change and created a dedicated task to schemas copies (when not part of `build` directly)